test/NuGet.Configuration.Test/NuGet.Configuration.Test.csproj

```diff
@@ -35,13 +35,17 @@
     <Reference Include="Ionic.Zip">
       <HintPath>..\..\packages\DotNetZip.1.9.3\lib\net20\Ionic.Zip.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.QualityTools.Testing.Fakes, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />
     <Reference Include="NuGet.Configuration, Version=1.0.0.0, Culture=neutral, PublicKeyToken=2e465378e3b1a8dd, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\packages\NuGet.Configuration.1.0.0-master-50072141\lib\net45\NuGet.Configuration.dll</HintPath>
+      <HintPath>..\..\src\NuGet.Configuration\obj\Debug\NuGet.Configuration.dll</HintPath>
```

Comment:
This reference got broken.  The reference should be removed though, and just use a project reference from the Configuration library into the test project.